embuilder.py

```diff
@@ -205,6 +194,9 @@ def main():
   if args.pic:
     settings.RELOCATABLE = 1
 
+  if args.pthreads:
+    settings.USE_PTHREADS = 1
```

Comment:
I don't think this works does it?   Does `./embuilder build libc --force --pthreads`  build the `-mt` flavor of libc for you locally?

You would need to add `-mt` suffixes to all the tasks I think, no?

I suggest reverting this part of the change to keep to small/simple and adding the new flags as followup (along with a test).  Actually without this part this change is NFC right?

Reply:
You are right, it does not work for system libraries like libc. I tested it only with ports like sqlite3. `./embuilder build sqlite3 --force --pthreads` creates `sqlite3-mt`. I'm dropping the `--pthreads` option from the PR and open a new one.

What does **NFC** stand for? Near field communication doesn't make sense here. :)

Reply:
> What does **NFC** stand for?

It stands for Non-Functional Change, as adopted from the LLVM lexicon:
https://llvm.org/docs/Lexicon.html#nfc

(The **MAINT**_enance_ abbreviation is somewhat similar, which is often used in Python projects)